numba/tests/test_array_reductions.py

```diff
@@ -992,7 +992,6 @@ def check_exception(self, pyfunc, msg):
         with self.assertRaises(ValueError) as e:
             cfunc(self.zero_size)
         self.assertIn(msg, str(e.exception))
-        self.disable_leak_check()
```

Comment:
Is this removal correct?

Reply:
This is from the revert. With the refcount pruner, the tested jit function here will not have retain any references.

Reply:
ah I see, thanks!